Packs/Mimecast/pack_metadata.json

```
@@ -2,7 +2,7 @@
     "name": "Mimecast",
     "description": "Mimecast unified email management offers cloud email services for email security, continuity and archiving emails. Read the detailed instructions to understand how to configure the integration's parameters.",
     "support": "xsoar",
-    "currentVersion": "2.1.10",
+    "currentVersion": "2.1.11",
```

```suggestion
    "currentVersion": "2.2.0",
```
Please change in the release notes as well

---

Done